### DISCUSSION TITLE
Solaredge Battery Discharge

### DISCUSSION THREAD

**Discussion by szadro** [amberelectric/public-api#143] [Q&A]
When in ‘maximize export’ mode my Solaredge battery is not sustaining 5kW … it is going through a constant cycling of:
•	Ramping up to 5kW – which takes ~60 seconds;
•	Holding 5kW for 60-90 seconds; 
•	Then going back to 0kW for ~5 seconds; and
•	Then the cycle repeats.
In Home Assistant I have noticed that the battery discharge limit is set to 5000W in "maximise export".   But in self consumption mode" the discharge limit is set to 11400W.

Is anyone else experiencing this issue?  This is really limiting the systems ability to take advantage of price spikes.

**Comment by purcell-lab** [comment]
I don't have a SolarEdge battery, but I did see an similar issue with my SolarEdge inverters and switching to zero export mode.

I would say a value and then a few moments later the default value would be set.

The cause ended up SmartShift was overriding my local settings every minute.

I had to get my SolarEdge inverters unenrolled from SmartShift, just switching SmartShift was insufficient.

I have a powerwall so have kept my battery enrolled in SmartShift.

As you are using SolarEdge for solar and battery you may not be able to uneroll from SmartShift without loosing core functionality.

**Comment by petehhhhh** [comment]
Hi,

i had this issue. Spent many hours sorting out. Two pieces to the fix. Check you have your input / export meter setup correctly. If not sure how to do so raise a ticket with Solaredge support. This helped but I still had issues. My ultimate fix was to move to a hardwired Ethernet connection to the inverter and swapping the lead inverter to be the one with the battery and meter. Has worked perfectly since.
